Dependencies updates errors

Markel opened the pull request:
The [job #106](https://travis-ci.com/github/MarkelFe/metacritic-crawler/jobs/292831128) is the last job without errors. [Job #107](https://travis-ci.com/github/MarkelFe/metacritic-crawler/jobs/295848384) shows the first error. Starting from [job #109](https://travis-ci.com/github/MarkelFe/metacritic-crawler/jobs/301006024) it derivated in a new error which triggered Travis CI to fail the cron job.

The objective is not to update to work in new technologies (although you are welcome to submit a pull request), it is to stabilize the script in old technologies (basically retiring the script)

Markel committed:
Update year

In the Readme

MarkelFe commented:
I leave here the requirements differences between the builds

|  | Build 106 | Build 107 | Build 109 |
|------------------|---------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| **Scrapy** | **1.8.0** | **2.0.0** | **2.0.1** |
| Tqdm | 4.43.0 | 4.43.0 | 4.43.0 |
| Six | 1.12.0 | 1.12.0 | 1.12.0 |
| Parsel | 1.5.2 | 1.5.2 | 1.5.2 |
| Queuelib | 1.5.0 | 1.5.0 | 1.5.0 |
| pyOpenSSL | 19.1.0 | 19.1.0 | 19.1.0 |
| cryptography | 2.8 | 2.8 | 2.8 |
| protego | 0.1.16 | 0.1.16 | 0.1.16 |
| cssselect | 1.1.0 | 1.1.0 | 1.1.0 |
| PyDispatcher | 2.0.5 | 2.0.5 | 2.0.5 |
| lxml | 4.5.0 | 4.5.0 | 4.5.0 |
| **Twisted** | 19.10.0 | **19.10.0** | **20.3.0** |
| zope.interface | 4.7.1 | 4.7.1 | 5.0.1 |
| w3lib | 1.21.0 | 1.21.0 | 1.21.0 |
| service-identity | 18.1.0 | 18.1.0 | 18.1.0 |
| cffi! | 1.14.0 | 1.14.0 | 1.14.0 |
| PyHamcrest | 2.0.1 | 2.0.2 | 2.0.2 |
| **Automat** | 20.2.0 | 20.2.0 | 20.2.0 |
| constantly | 15.1.0 | 15.1.0 | 15.1.0 |
| incremental | 17.5.0 | 17.5.0 | 17.5.0 |
| **attrs** | 18.2.0 | 18.2.0 | 18.2.0 |
| hyperlink | 19.0.0 | 19.0.0 | 19.0.0 |
| setuptools | 40.6.3 | 40.6.3 | 40.6.3 |
| pyasn1 | 0.4.8 | 0.4.8 | 0.4.8 |
| pyasn1-modules | 0.2.8 | 0.2.8 | 0.2.8 |
| pycparser | 2.19 | 2.20 | 2.20 |
| idna | 2.9 | 2.9 | 2.9 |
| Notes: | automat 20.2.0 has requirement attrs>=19.2.0, but you'll have attrs 18.2.0 which is incompatible. | automat 20.2.0 has requirement attrs>=19.2.0, but you'll have attrs 18.2.0 which is incompatible. | automat 20.2.0 has requirement attrs>=19.2.0, but you'll have attrs 18.2.0 which is incompatible.<br>twisted 20.3.0 has requirement attrs>=19.2.0, but you'll have attrs 18.2.0 which is incompatible. |

Markel committed:
Stop scrapy and twisted

Those seem the ones that were causing the problems mainly

Markel committed:
Add attrs

That will remove the *automat 20.2.0 has requirement attrs>=19.2.0, but you'll have attrs 18.2.0 which is incompatible*

Markel committed:
Attrs auto-update

Change from = to > so it updates with the power of the internet

Markel committed:
Update readme

Just leave a note that we have outdated packages
